.github/workflows/auto-merge.yml

```
@@ -1,44 +1,192 @@
+                shouldMerge = true;
+                console.log('✅ Generic bump detected - treating as patch update');
+              }
+            }
+            
+            // Check if all required checks are passing
+            const { data: checks } = await github.rest.checks.listForRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: pr.head.sha
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)